### PR TITLE
appending _pre_launch to all shadow org handles

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

**Comment by eishay** [comment]
@ryanpbrewster please review

File: kifi-backend/modules/shoebox/app/com/keepit/model/OrganizationRepo.scala [hunk 1]
```diff
@@ -73,6 +75,11 @@ class OrganizationRepoImpl @Inject() (
     q.list
   }
 
+  def getShadowOrgs()(implicit session: RSession): Seq[Organization] = {
+    val q = for { row <- rows if row.ownerId === AdminOrganizationController.fakeOwnerId } yield row
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/model/OrganizationRepo.scala]: Active orgs only

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminOrganizationController.scala [hunk 2]
```diff
@@ -373,6 +373,21 @@ class AdminOrganizationController @Inject() (
     Redirect(com.keepit.controllers.admin.routes.AdminOrganizationController.organizationViewBy(orgId))
   }
 
+  def updateShadowOrgHandles() = AdminUserPage { implicit request =>
+    val orgs = db.readOnlyReplica { implicit s =>
+      orgRepo.getShadowOrgs()
+    }
+    orgs foreach { org =>
+      db.readWrite { implicit session =>
+        assert(org.ownerId == fakeOwnerId)
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminOrganizationController.scala:383]: As a general rule for these types of migrations I think it's best to grab just the ids, then use the I'd to retrieve the model right before you modify it

**Comment by ryanpbrewster** [comment]
This looks good to me, as long as you filter out inactive orgs

**Comment by eishay** [comment]
sure

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminOrganizationController.scala (hunk 2 above)
Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminOrganizationController.scala:383]: sure, done

**Pull request merged**